unicodetools/src/main/resources/org/unicode/text/UCD/AdditionComparisons/198.txt

```diff
@@ -0,0 +1,18 @@
+# Symbol: Saudi riyal sign
+# https://github.com/unicode-org/utc-release-management/issues/198
+
+# Names always differ.
+# Age always differs since these tests are comparing additions to pre-existing characters.
+Ignoring Name Age:
+
+# Ignore the security and IDNA properties, as these are not yet included for provisionally assigned characters.
+Ignoring Confusable_MA Identifier_Status Identifier_Type Idn_Status Idn_Mapping Idn_2008:
```

Comment:
It's accepted, not provisionally assigned.
Should we just set the ID_Type etc. now?

Reply:
I don’t want to go there for the addition comparison tests for now.

Also, I do not know how to turn the security crank.